Cargo.lock

```diff
@@ -709,7 +709,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "117725a109d387c937a1533ce01b450cbde6b88abceea8473c4d7a85853cda3c"
 dependencies = [
  "lazy_static",
- "windows-sys 0.59.0",
+ "windows-sys 0.48.0",
```

Comment:
I wonder why this has gone even further back 🤔

Reply:
It actually depends on `windows-sys = ">=0.48,<=0.59"`. I don't know why cargo chooses an older version either...

Reply:
Fixed:)